README.md

```diff
@@ -1,202 +1,33 @@
-![Houston logo](https://storage.googleapis.com/gcs-callhouston-asset/houston-title.png)
+# Houston
 
 ![PyPI](https://img.shields.io/pypi/v/houston-client)
 
-Houston Python Client Library (https://callhouston.io), links components to a simple workflow api.
+Python Client for Houston (https://callhouston.io). 
+Houston is an open source tool for building low cost, scalable, platform-agnostic data pipelines. 
+Teams can share data processes as microservices using a simple API.
 
 
 ## Installation
 
-This client can be easily installed via pip:
+This client can be installed via pip:
 
-```commandline
+```bash
 pip install houston-client
 ```
 
-## Requirements
-
-- Python >= 3.7
-
-## Proxy
-
-Should you need to use a proxy for your Houston requests, please set them as an environment variable:
+If you're running Houston on Google Cloud Platform you will need the 'gcp' plugin:
 
 ```bash
-# windows
-set https_proxy=10.0.0.1 
-
-# linux
-export https_proxy=10.0.0.1
-
+pip install "houston-client[gcp]"
 ```
 
 ## Usage
 
-Please read the documentation before getting started: https://callhouston.io/docs
```

Comment:
I feel the README.md would benefit from outlining the functionality areas in this client (used in Python and as a Cloud Function wrapper)

Reply:
I've added an overview to the README and linked to various new documentation articles